pkg/cmd/scan/output/assets/index.tmpl

```diff
@@ -11,7 +11,7 @@
 <div class="container">
     <header>
         <div class="div-left">
-            {{ .LogoSvg }}
+            <div id="brand_logo">{{ .LogoSvg }}</div>
```

Comment:
Not a huge fan of using id just for css purposes. Any other reasons you are using an id here?

Reply:
Yes, the purpose is to make this block unique so we can replace it in other tools.